ui/components/HelmChartDetail.tsx

```diff
@@ -34,6 +34,8 @@ function HelmChartDetail({ className, helmChart }: Props) {
       info={[
         ["Type", Kind.HelmChart],
         ["Chart", helmChart.chart],
+        ["Version", helmChart.version],
+        ["Revision", helmChart.revision],
```

Comment:
:bulb: The "version" vs "revision" distinction is weird to me. At least to me, "version" and "revision" can be used interchangeably, and I'd use revision for "automatic" versioning (i.e. git commits) and version for "active" versioning (i.e. git tags) - and that's not at all what's going on here! Version is a "version mask", "version selector" or "version specification", and revision is "actual version" or "highest version matching the version field". I looked at the flux documentation to see if they spell it out more, but the best they're doing is "artifact revision", which is perfectly accurate but also requires you to have a deep understanding of the source controller for it to be helpful.

This isn't too bad on the helm release page (because "last applied" and "last attempted" helps explain that they're "discovered" and thus status, so that helps suggest "chart version" isn't discovered and thus specified), but here I can't help go "wait, what?".

**tl;dr**: My concrete suggestion is to change "Revision" to "Latest Revision" or "Current Revision". That way it would also "look like" it comes from status, to help distinguish between the two.